website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -74,83 +74,96 @@ The following Kubernetes resource annotations could be used on a pod to control
+        
         ```yaml
         annotations:
           "consul.hashicorp.com/connect-service-upstreams":"[service-name].[service-namespace]:[port]:[optional datacenter]"
         ```
+
         If the namespace is not specified, the annotation defaults to the namespace of the source service.
-        If you are not using Consul Enterprise 1.7+, Consul interprets the value placed in the namespace position as part of the service name.
+        If your version of Consul Enterprise in lower than v1.7, Consul interprets the value placed in the namespace position as part of the service name.
 
       - Admin partitions (requires Consul Enterprise 1.11+): Upstream services may be running in a different
-        partition. You must specify the namespace when specifying a partition. Place the partition name after the namespace. If you specify the name of the datacenter (optional), it must be the local datacenter. Communicating across partitions using this method is only supported within a
-        datacenter. For cross partition communication across datacenters, refer to [cluster
-        peering](/consul/docs/k8s/connect/cluster-peering/tech-specs).
+        partition. When specifying a partition, you must also specify a namespace. Place the partition name after the namespace. If you specify the name of the datacenter, it must be the local datacenter. Communicating across partitions using this method is only supported within a
+        datacenter. For cross partition communication across datacenters, [establish a cluster
+        peering connection](/consul/docs/k8s/connect/cluster-peering/usage/establish-peering) and set the upstream with a labeled annotation format.
+
         ```yaml
         annotations:
           "consul.hashicorp.com/connect-service-upstreams":"[service-name].[service-namespace].[service-partition]:[port]:[optional datacenter]"
         ```
-    - [Prepared queries](/consul/api-docs/query): Prepend the annotation
-      with `prepared_query` and place the name of the query at the beginning of the string.
-      ```yaml
-      annotations:
-        'consul.hashicorp.com/connect-service-upstreams': 'prepared_query:[query name]:[port]'
-      ```
+        
+      - Prepared queries: To configure a [prepared query](/consul/api-docs/query) in an upstream annotation, prepend the annotation
+        with `prepared_query` and then invoke the name of the query.
+
+        ```yaml
+        annotations:
+          'consul.hashicorp.com/connect-service-upstreams': 'prepared_query:[query name]:[port]'
+        ```
+
+    - **Labeled** (requires Consul for Kubernetes v0.45.0+):
+
+      The labeled annotation format is required when specifying an upstream service in a different datacenter. You can specify a peer, namespace, partition, or datacenter, but the format supports only one peer, datacenter, or partition per annotation.
```

Comment:
If we change the text above to generally recommend the labeled format, then it seems unnecessary to say that this is required when specifying an upstream in a different datacenter. It also suggests that unlabeled formats can't reference upstreams in a different datacenter---they can, as long as it is a WAN federated datacenter (not a cluster peer).

I'm not sure what the original text of "supports only one peer, datacenter, or partition" means. I think it's intended to mean that you can only specify one of those fields per upstream. I worry that the wording of "per annotation" can give a false impression that the `connect-service-upstreams` annotation can only have one upstream.

Perhaps something like:
```suggestion
      The labeled annotation format allows you to reference any service as an upstream. You can specify a Consul Enterprise namespace. You can also specify an admin partition in the same datacenter, a cluster peer, or a WAN federated datacenter.
```